apps/contrib/templates/a4_candy_contrib/includes/form_field.html

```diff
@@ -8,15 +8,21 @@
     </label>
     {% block help_text %}
     {% if field.help_text %}
-    <div class="form-hint">
+    <p class="form-hint" id="{{ field.id_for_label }}_hint">
         {{ field.help_text }}
-    </div>
+    </p>
     {% endif %}
     {% endblock %}
     {% block field %}
         <div class="widget widget--{{ field|widget_type }}">
+          {% if field.errors %}
+            {{ field|attr:"aria-invalid=true" }}
```

Comment:
it would be good to also have `aria-describedby="{{field.id_for_labe }}_hint` but wasn't sure how